README.md

```diff
@@ -2,17 +2,72 @@
 
 ## About ##
 
-This is an attempt to build just another web-crawler, also called _Spider_. Its purpose is to get data from a website in a form of a list of objects. This data can be then used to download linked files and place them in a database or a folder structure.
+Onigumo je web-crawler.
+
+Onigumo "prolézá" webové stránky či aplikace. Jejich obsah a případná metadata uloží do strukturované podoby, která je vhodná pro další strojové zpracování.
 
 ## Architecture ##
 
-### Building blocks ###
+Onigumo je rozděleno do tří na sebe vzájemně navazujících částí:
+
+* operator - řízení
+* downloader - stahování dat
+* parser - zpracování dat
```

Comment:
```suggestion
* operator – řízení
* downloader – stahování dat
* parser – zpracování dat
```

Reply:
I can not write long hyphen on my keyboard, is there any trick how to find out how to write on keyboard please?